pkg/certs/cert-inspection/certgraphanalysis/metadata_options.go

```diff
@@ -149,6 +152,35 @@ var (
 			return timestampReg.ReplaceAllString(path, "<timestamp>.pem")
 		},
 	}
+	HumanizeRewritePeriod = &metadataOptions{
```

Comment:
ok, and how is this var being used ? by external tools that pull this library ?

Reply:
Yeah, lets go with `Rewrite` - its more similar to what other functions are named